src/Roave/SecurityAdvisories/VersionConstraint.php

```diff
@@ -159,6 +159,18 @@ public function mergeWith(self $other) : self
         ));
     }
 
+    public static function sort(self $a, self $b): int
```

Comment:
Is this method needed on this particular class? Looks like completely external?

Reply:
Nope, I put it there because it sorts only between elements of that class. If you prefer a separated class with `__invoke`, I'll do it! :+1:

Reply:
Done!